options.go

```diff
@@ -70,6 +73,9 @@ type Options struct {
 	// potential overrides.
 	Provider          string `flag:"provider" cfg:"provider"`
 	OIDCIssuerURL     string `flag:"oidc-issuer-url" cfg:"oidc_issuer_url"`
+	SkipOidcDiscovery bool   `flag:"skip-oidc-discovery" cfg:"skip_oidc_discovery"`
```

Comment:
Can we capitalise OIDC to be consistent
```suggestion
	SkipOIDCDiscovery bool   `flag:"skip-oidc-discovery" cfg:"skip_oidc_discovery"`
```

Reply:
agreed, have changed it